drakcore/drakcore/frontend/src/AnalysisMain.js

```diff
@@ -220,12 +233,52 @@ class AnalysisMain extends Component {
       );
     }
 
+    let metadata;
+    if (this.state.metadata) {
+      metadata = (
+        <div background>
+          <table className="table table-striped table-bordered">
+            <tbody>
+              <tr>
+                <td>Sha256</td>
+                <td>{this.state.metadata.sample_sha256}</td>
+              </tr>
+              <tr>
+                <td>Magic bytes</td>
+                <td>{this.state.metadata.magic_output}</td>
+              </tr>
+              <tr>
+                <td>Start command</td>
+                <td>{this.state.metadata.start_command}</td>
+              </tr>
+              <tr>
+                <td>Started at</td>
+                <td>{formatTimestamp(this.state.metadata.time_started)}</td>
+              </tr>
+              <tr>
+                <td>Finished at</td>
+                <td>{formatTimestamp(this.state.metadata.time_finished)}</td>
+              </tr>
+            </tbody>
```

Comment:
We could replace this with "react for", but then labels wouldn't be as nice as they are now.

Reply:
:bipolar: this is not that big yet, both options sound OK